heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/index/SingleIndexMapping.java

```diff
@@ -24,34 +24,42 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableList;
+import java.util.HashMap;
```

Comment:
wdyt about making this a kotlin data class too given `RotatingIndexMapping` was converted?

Reply:
sure, I initially wasn't touching it and then adding the settings in last minute. I'll convert it